pyccl/cosmology.py

```diff
@@ -101,6 +103,12 @@ def _make_methods(cls=None, *, modules=_TOP_LEVEL_MODULES, name=None):
     return cls
 
 
+@dataclass
+class CosmologyData:
```

Comment:
Can you add a comment here for future developers that the aim is for this class to be used to port other background calculators from C to python?

Also, we should make it a private class (`_CosmologyData`), and perhaps relabel it to `_CosmologyBackgroundData` to make it clearer what it's supposed to hold.